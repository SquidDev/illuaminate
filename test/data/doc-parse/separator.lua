----------------------
-- This block should not be considered as a doc-comment
----------------------

------ This one should be

---
-- This one also should be
