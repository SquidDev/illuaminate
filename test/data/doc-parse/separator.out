((source separator.lua[5:1-5:25])
 (description "\\--- This one should be\n\n"))
((source separator.lua[7:1-8:26])
 (description "This one also should be\n\n"))
